spec/helpers/form_helper_spec.rb

```diff
@@ -7,4 +7,19 @@
       helper.list_of_organisations
     end
   end
+
+  describe "#list_of_user_roles" do
```

Comment:
I'm not sure if this test is necessary..?

Reply:
This is something I find difficult to make calls on, so I can learn, would you mind expanding on why you feel like this isn't necessary?